docs/update_and_migration/from_3.3/to_4.0.md

```diff
@@ -168,6 +168,53 @@ php bin/console ibexa:migrations:migrate
 
 ## Update your custom code
 
+### GraphQL
+
+Some property names has changed and your GraphQL queries need to be updated.
+
+| 3.3 name                     | 4.0 name                  |
+|:-----------------------------|:--------------------------|
+| `id`                         | `contentId`               |
```

Comment:
```suggestion
| `id`                         | `contentId`               |
```
Can we mention that it's an argument to the `DomainContent` object?

The query below is still valid, it's not a simple search and replace 😢 
```
{
  content {
    folder(contentId: 1) {
      	id
       _contentInfo{
        name
      }
    }
  }
}
```